console/src/components/Services/EventTrigger/EventActions.js

```diff
@@ -107,21 +106,14 @@ const loadPendingEvents = () => (dispatch, getState) => {
             '*',
             { name: 'logs', columns: ['*'], order_by: ['-created_at'] },
           ],
-          where: { delivered: false, error: false, tries: 0 },
+          where: { delivered: false, error: false, tries: 0, archived: false },
```

Comment:
What is this?

Reply:
This was some event trigger server refactor related thing. Event data can be archived so we need to add thid flag to ensure we fetch only non archived data

Reply:
it was initially added after version check below